relationships/synthesis/EXT-SERVICE-to-INFRA-KINESISSTREAM.yml

```diff
@@ -0,0 +1,22 @@
+relationships:
+  - name: extServiceCallsInfraKinesisStream
+    version: "1"
+    origins:
+      - Distributed Tracing
+    conditions:
+      - attribute: eventType
+        anyOf: [ "Span" ]
+      - attribute: rpc.service
+        anyOf: [ "kinesis" ]
```

Comment:
If we're primarily receiving data in spans, i guess we should opt for:
`origins:
      - Distributed Tracing `
      instead of 
   `origins:
      - Opentelemetry`
 Currently, no pull request exists for APM to AWS Kinesis streaming. We anticipate that APM data will continue to arrive in span format; if it's in metrics, we'll use APM Metrics. Do we need to wait for the APM to AWS Kinesis stream PR?